NEWS.md

```diff
@@ -47,6 +47,12 @@ Language changes
   used to yield the string " a\nb", since the single space before `b` set the indent level.
   Now the result is "a\n b", since the space before `b` is no longer considered to occur
   at the start of a line. The old behavior is considered a bug ([#35001]).
+
+* `<:` and `>:` can now be broadcasted over arrays with `.<:` and `.>:`  ([#35085])
+
+* Color now defaults to on when stdout and stderr are TTYs ([#34347])
```

Comment:
Delete; this has been moved to the command-line section below.

Reply:
Removed in eb6df88